plugin-dev/Sentry.uplugin

```diff
@@ -14,7 +14,7 @@
 	"IsBetaVersion": false,
 	"IsExperimentalVersion": false,
 	"Installed": false,
-	"Modules": [
+	"modules": [
```

Comment:
I believe we should leave `Modules` here since it is a pre-determined field for the plugin descriptor file (https://docs.unrealengine.com/5.0/en-US/API/Runtime/Projects/FPluginDescriptor/Modules/)

Reply:
except these are not "Modules" in Unreal world? https://docs.unrealengine.com/4.27/en-US/ProgrammingAndScripting/ProgrammingWithCPP/Modules/

Reply:
These are exactly the same "Modules". Probably my previous comment was a bit misleading - I've meant we should leave "Modules" instead of "modules" (all lowercase) to keep things consistent here

Reply:
my bad, I was referring to the repo/modules with gitmodules. this line is a mistake of course

```suggestion
	"Modules": [
```